spinalcordtoolbox/deepseg/models.py

```diff
@@ -69,7 +73,8 @@ def is_valid(path_model):
     :param path_model: str: Absolute path to folder that encloses the model files.
     """
     name_model = path_model.rstrip(os.sep).split(os.sep)[-1]
-    return os.path.exists(os.path.join(path_model, name_model + '.pt')) and \
+    return (os.path.exists(os.path.join(path_model, name_model + '.pt')) or
+           os.path.exists(os.path.join(path_model, name_model + '.onnx'))) and \
```

Comment:
The priority is given to onnx file in segment_volume, so I agree with your change here ;)